src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBranch.java

```diff
@@ -149,6 +149,10 @@ public PiModel getPiModel() {
         return piModel;
     }
 
+    public PiModel getPiModel(int tapPosition) {
```

Comment:
Why did you add this method ? It is never used.

Reply:
Solved in another comment.